module-system/utils/sov-data-generators/src/bank_data.rs

```diff
@@ -0,0 +1,131 @@
+use std::rc::Rc;
+
+use sov_bank::{get_token_address, Bank, CallMessage, Coins};
+use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_api::utils::generate_address;
+use sov_modules_api::{Context, EncodeCall, Module, PrivateKey, Spec};
+
+use crate::MessageGenerator;
+
+pub struct TransferData<C: Context> {
+    pub sender_pkey: Rc<DefaultPrivateKey>,
```

Comment:
Question: we use the address as generic but the private key as concrete. Any reason to do so? Shouldn't both follow the same pattern? (i.e. both be either generic or concrete)